<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>511</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="KeyTab">
       <attribute name="title">
        <string>Private Keys</string>
       </attribute>
       <layout class="QHBoxLayout" name="layout_2">
        <item>
         <widget class="KeyTreeView" name="keyView"/>
        </item>
        <item>
         <widget class="QFrame" name="keyButtons">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="BNnewKey">
             <property name="text">
              <string>&amp;New Key</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BNexportKey">
             <property name="text">
              <string>&amp;Export</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BNimportKey">
             <property name="text">
              <string>&amp;Import</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BNimportPFX">
             <property name="text">
              <string>Import PFX (PKCS#12)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BNdetailsKey">
             <property name="text">
              <string>&amp;Show Details</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BNdeleteKey">
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="bigKey">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>94</height>
              </size>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/KeyTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BNdeleteKey</sender>
   <signal>clicked()</signal>
   <receiver>keyView</receiver>
   <slot>deleteItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BNdetailsKey</sender>
   <signal>clicked()</signal>
   <receiver>keyView</receiver>
   <slot>showItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BNimportKey</sender>
   <signal>clicked()</signal>
   <receiver>keyView</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BNnewKey</sender>
   <signal>clicked()</signal>
   <receiver>keyView</receiver>
   <slot>newItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BNexportKey</sender>
   <signal>clicked()</signal>
   <receiver>keyView</receiver>
   <slot>exportItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
